gh_pages/_source/session1/ros2/3-Creating-a-ROS-Package-and-Node.md

```diff
@@ -161,29 +162,29 @@ Your goal is to create your first ROS node:
 
    ``` cmake
    add_executable(vision_node src/vision_node.cpp)
    ament_target_dependencies(vision_node rclcpp)
    ```
 
-   These lines should be place after the section labeled `# find dependencies`. The flow of a _CMakeLists.txt_ file is almost always to first find all dependencies the current project requires, then to declare the _targets_ the current project is creating, and finally to specify extra information such as how to test the targets and things that projects making use of this one need to know. Note that the term "project" is used here in the context of CMake. Each ROS package acts as a separate CMake project (notice the `project(myworkcell_core)`) line at the very top of the file). 
+   These lines should be place after the section labeled `# find dependencies` and before the `ament_package()` call. The flow of a _CMakeLists.txt_ file is almost always to first find all dependencies the current project requires, then to declare the _targets_ the current project is creating, and finally to specify extra information such as how to test the targets and things that projects making use of this one need to know. Note that the term "project" is used here in the context of CMake. Each ROS package acts as a separate CMake project (notice the `project(myworkcell_core)`) line at the very top of the file). 
 
-   _Note: You're allowed to spread most of the CMakeLists rules across multiple lines, which is often required when a target contains many source files or has many dependencies._
+  * See [ament_cmake documentation](https://docs.ros.org/en/foxy/How-To-Guides/Ament-CMake-Documentation.html) for more details on common ROS2 CMakeLists rules.
+  * _You're allowed to spread most of the CMakeLists rules across multiple lines, which is often required when a target contains many source files or has many dependencies._
 
 1. We've now told CMake about the _vision_node_ executable and how to build it, but to actually run it, the file must be *installed* along with all the other workspace outputs. Typically, the installation location will be the `install/` directory alongside the `src/` directory. Add the following lines to declare an installation rule for the _vision_node_ executable:
 
    ``` cmake
-   # Mark executables and/or libraries for installation
+   # Mark executables for installation
    install(TARGETS vision_node
-       ARCHIVE DESTINATION lib/${PROJECT_NAME}
-       LIBRARY DESTINATION lib/${PROJECT_NAME}
-       RUNTIME DESTINATION lib/${PROJECT_NAME}
-   )
+       DESTINATION lib/${PROJECT_NAME})
    ```
+   
+   * _ROS2 installs both executable programs and libraries into the `lib` directory_
```

Comment:
Might note libraries go in `lib` but executables are supposed to be package-specific subdirs of `lib/`, which is the only place `ros2 run` etc will find them.

Reply:
Made an update in upcoming commit.